src/Chr.Avro.Binary/Serialization/BinaryReader.cs

```diff
@@ -52,6 +52,18 @@ public byte[] ReadBytes()
             return ReadFixed((int)ReadInteger());
         }
 
+        /// <summary>
+        /// Reads variable-length binary data from the current position and advances the reader.
+        /// </summary>
+        /// <returns>
+        /// An span of <see cref="byte" />s with length specified by the integer at the current
+        /// position.
+        /// </returns>
+        public ReadOnlySpan<byte> ReadSpan()
```

Comment:
For consistency with `ReadFixedSpan`:

```suggestion
        public ReadOnlySpan<byte> ReadBytesSpan()
```

These names correspond to the `"bytes"` and `"fixed"` Avro types